app/views/lookup_keys/_values.html.erb

```diff
@@ -0,0 +1,23 @@
+<table class="table table-condensed">
+  <thead>
+  <tr>
+    <th colspan='2' class='col-md-5'><%= _('Attribute Type') %> <span class="help-inline"><%= popover("", _("Matcher is a combination of an attribute and its value, if they match, the value below would be provided.<br> You may use any attribute foreman knows about, such as facts etc for example: <code> domain = example.com </code> or <code> is_virtual = true</code>"),
+                                                                :title => _("Explain matchers")).html_safe %></span></th>
+    <th class='col-md-4'><%= _('Value') %></th>
+    <th class='col-md-3'><%= _('Use Puppet Default') %><span class="help-inline"> <%= use_puppet_default_help() %></span></th>
```

Comment:
When I tried to change it to `col-md-2` it became a two line header.
Maybe if I change it to 'Puppet default' that would be enough?